src/cyw43_ll.c

```diff
@@ -1827,13 +1829,19 @@ int cyw43_ll_bus_init(cyw43_ll_t *self_in, const uint8_t *mac) {
 /*******************************************************************************/
 // WiFi stuff
 
-static void cyw43_do_ioctl_u32(cyw43_int_t *self, uint32_t kind, uint32_t cmd, uint32_t val, uint32_t iface) {
+static void cyw43_set_ioctl_u32(cyw43_int_t *self, uint32_t cmd, uint32_t val, uint32_t iface) {
     uint8_t *buf = &self->spid_buf[SDPCM_HEADER_LEN + 16];
     cyw43_put_le32(buf, val);
-    cyw43_do_ioctl(self, kind, cmd, 4, buf, iface);
+    cyw43_do_ioctl(self, SDPCM_SET, cmd, 4, buf, iface);
+}
+
+static uint32_t cyw43_get_ioctl_u32(cyw43_int_t *self, uint32_t cmd, uint32_t iface) {
+    uint8_t *buf = &self->spid_buf[SDPCM_HEADER_LEN + 16];
+    cyw43_put_le32(buf, 0);
+    cyw43_do_ioctl(self, SDPCM_GET, cmd, 8, buf, iface);
```

Comment:
hmm, yes. Not sure what that cyw43_put_le32 thinks it's doing either.

Reply:
The put_le32 makes sense, to clear the initial value in case the SDPCM_GET command reads it (although probably it's not used, but safer to zero it out like this).

Reply:
Should be fixed now